builtin/credential/okta/path_users.go

```diff
@@ -0,0 +1,139 @@
+package okta
```

Comment:
We did some internal discussion and per-user policies is also desired. So the full combination is policies from set of groups the user belongs to + policies attached to the specific user.